server/blob/blob-cassandra/src/main/java/org/apache/james/blob/cassandra/cache/CachedBlobStore.java

```diff
@@ -243,4 +252,9 @@ private boolean isAbleToCache(byte[] bytes) {
     private Mono<byte[]> readBytesFromBackend(BucketName bucketName, BlobId blobId) {
         return Mono.from(backend.readBytes(bucketName, blobId));
     }
+
+    private <T> T increaseCacheMetricCount(T data) {
+        metricRetrieveMissCount.increment();
```

Comment:
Wouldn't it be easier to record miss counts in CassandraBlobStoreCache (like you did for hit?)